src/middlewared/middlewared/plugins/smart.py

```diff
@@ -389,7 +389,7 @@ async def manual_test(self, disks):
                 else:
                     verrors.add(
                         f'disks.{index}.identifier',
-                        f'{disk["identifier"]} is not valid. Please provide a valid disk identifier.'
+                        f'{disk["identifier"]} does not support S.M.A.R.T test.'
```

Comment:
In case an invalid identifier is supplied, we should raise an appropriate message that the supplied identifier is invalid. So let's please have this separately for the case where the disk identifier is valid but the disk does not support SMART test.